packages/core/src/browser/keybinding.ts

```diff
@@ -139,9 +140,9 @@ export class KeybindingRegistry {
         }
     }
 
-    registerKeybindings(...bindings: Keybinding[]): void {
+    registerKeybindings(bindings: Keybinding[], scope?: KeybindingScope): void {
```

Comment:
please leave `registerKeybindings` and `registerKeybinding` as it is to avoid api breaking

introduce protected methods and call them which accept the scope, like `doRegisterKeybindings` and `doRegisterKeybinding `

Reply:
adding an optional parameter to `registerKeybinding` (taking one binding at once) doesn break the API, should I still create a second one ?

Reply:
but i don't think we need scopes for keybindings, it is keymaps related, programmatic keybindings should go to the default scope always.

Reply:
I see now, ok !